debian.sh

```
@@ -237,7 +239,7 @@ randomize_maint_mysql_pass() {
   UPDATE user SET password=PASSWORD(\""$PMAPASS"\") WHERE user='phpmyadmin';\nFLUSH PRIVILEGES;"
   echo -e "$MYSQLCOMMAND" > "$FOLD/hdd/etc/mysql/pwchange.sql"
   execute_chroot_command "/etc/init.d/mysql start >>/dev/null 2>&1"
-  execute_chroot_command "mysql --defaults-file=/etc/mysql/debian.cnf < /etc/mysql/pwchange.sql >>/dev/null 2>&1"; EXITCODE=$?
+  execute_chroot_command "mysql --defaults-file=/etc/mysql/debian.cnf < /etc/mysql/pwchange.sql >>/dev/null 2>&1"; declare -i EXITCODE=$?
```

Do you really want to throw the error messages away?